test/collapsers/javascript.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+var assert = require('power-assert');
+var Bluebird = require('bluebird');
+var describe = require('mocha').describe;
+var it = require('mocha').it;
+var collapser = require('../../lib/collapsers/javascript');
+
+describe('JavaScript collapser', function () {
+  it('should minify JavaScript', function () {
```

Comment:
At the very least you can ensure that size became smaller and/or test that e.g. names are being mangled (like `(function () { var smth; })()` shouldn't contain `smth` in the output).